daprdocs/content/en/reference/cli/dapr-run.md

```diff
@@ -36,7 +36,7 @@ dapr run [flags] [command]
 | `--dapr-http-port`, `-H`       | `DAPR_HTTP_PORT`     | `3500`                                                                             | The HTTP port for Dapr to listen on                                                                  |
 | `--enable-profiling`           |                      | `false`                                                                            | Enable "pprof" profiling via an HTTP endpoint                                                        |
 | `--help`, `-h`                 |                      |                                                                                    | Print the help message                                                                              |
-| `--run-file`, `-f`                 |                      |  Linux/MacOS: `$HOME/.dapr/dapr.yaml`                              | Run multiple applications at once using a Multi-App Run template file. Currently in [alpha]({{< ref "support-preview-features.md" >}}) and only available in Linux/MacOS                                                                     |
+| `--run-file`, `-f`                 |                      |  Linux/MacOS: `$HOME/.dapr/dapr.yaml`                              | Run multiple applications at once using a Multi-App Run template file. Currently in [alpha]({{< ref "support-preview-features.md" >}}) and only available in Linux/MacOS. Pass `-` as the value to read template from stdin.                                                                     |
```

Comment:
```suggestion
| `--run-file`, `-f`                 |                      |  Linux/MacOS: `$HOME/.dapr/dapr.yaml`                              | Run multiple applications at once using a Multi-App Run template file. Currently in [alpha]({{< ref "support-preview-features.md" >}}) and only available in Linux/MacOS. 

Pass `-` as the value to read template from stdin.                                                               |
```

Reply:
I would add an example here to be explicit. Having an example then makes it clear.

Reply:
@stuartleeks - Did you see my comment above ^^

Reply:
@msfussell  - I've updated to include an example of the full command syntax inline in the table. Is that what you were meaning?